src/test/java/net/snowflake/ingest/TestUtils.java

```diff
@@ -317,4 +322,27 @@ public static SimpleIngestManager getManagerUsingBuilderPattern(
         .setUserAgentSuffix(userAgentSuffix)
         .build();
   }
+
+  /**
+   * Given a channel and expected offset, this method waits until the last committed offset is equal
```

Comment:
Let's document how long we intend to wait.

Reply:
Fixed